### DISCUSSION TITLE
MAME-AO Discussion

### DISCUSSION THREAD

**Discussion by sam-ludlow** [sam-ludlow/mame-ao#1] [General]
You may talk openly about MAME-AO here.

For problems, errors, issues please go here https://github.com/sam-ludlow/mame-ao/issues

**Comment by paulmortlock** [comment]
How to make this cross platform windows and linux. I may trying it with Lutris and Wine.

My nirvana would be having this built into mame and i can have one place to store my mame files as I have a RPI4 and a Linux laptop and a windows laptop that would all be in sync and use my USB HD drives for portability. I am a .net developer and would like to help. Maybe update the html frontend.

**Reply by sam-ludlow** [reply]
Yeah I've been thinking about this.

I believe @cewatkins has had it running on WINE.

I made it framework 4.8 so it will run on most Windows boxes without requiring any other dependencies, like .net runtimes.

It should be pretty simple to port to current .net and run natively on anything that supports it like Linux and Pi.

The UI is just JS so should run on any browser. I didn't use any JS frameworks partly as an fun exercise, I don't really know or like any, and to keep it light weight.

But anyone could make an alternate UI the REST API is all in place ready to consume by anything.

Cheers
Sam

**Comment by sam-ludlow** [comment]
Closing - Cross platform is in an issue.
